__tests__/task12-get-api-articles-article-id-comment-count.test.js

```diff
@@ -0,0 +1,59 @@
+const { app, request, db, seed, data } = require("../testIndex");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("GET /api/articles/:article_id (comment_count)", () => {
+  test("should also now include comment_count", () => {
+    return request(app)
+      .get("/api/articles/1?comment_count")
```

Comment:
We shouldn't have to specificy this as a query. It should just be added as a property on our existing path /api/articles/:article_id. It should always be added to the data here.